### ISSUE TITLE
Adjust On-Boarding Flow - Phase In

### ISSUE THREAD

**Issue by ismu1** [sanctuarycomputer/smalls#60]
Adjust on-boarding flow so that body type and weight only appear after someone has selected their cats activity level

Many users get confused at the body type, weight and activity level section of the on-boarding flow (https://cl.ly/2O0b1Y3u411E). They forget to use the drop down for activity level, enter the weight and body type information and then are confused why they can't move forward.

To help reduce this confusion, we were hoping to adjust the way that things appeared.

Basically, what we want to do is after you put in ____ currently eats [dry food] we only want to reveal the activity level question.

After someone answers the activity level question, we'll reveal "he weighs about [13] lbs and his body looks like. Ideally this could fade in.
